modules/varnish/templates/default.vcl

```diff
@@ -281,99 +154,98 @@ sub recv_purge {
 	}
 }
 
-sub mw_vcl_recv {
-	call mw_rate_limit;
-	call mw_identify_device;
-
-	if (
+# Main MediaWiki Request Handling
+sub mw_request {
+	call rate_limit;
+	call mobile_detection;
+	
+	# Assigning a backend
+	if (req.http.X-Miraheze-Debug ~ "^(mw|test|mwtask)[0-9]{3}\.miraheze\.org$") {
+		# mw101.miraheze.org -> mw101, test101.miraheze.org -> test101, mwtask101.miraheze.org -> mwtask101
+		set req.backend_hint = regsub(req.http.X-Miraheze-Debug, "^((mw|test)[0-9]{3})\.miraheze.org", "\1");

```

Comment:
Needs to have mwtask here as well.

Reply:
It is since 4a70cc7. Are you checking an old commit.

Reply:
Done in 1ad9ac5